consensus_test.go

```diff
@@ -779,13 +779,13 @@ func TestRoundChange_PropertyMajorityOfVotingPowerAggreement(t *testing.T) {
 		validators := make(ValStringStub, numberOfNodes)
 		votingPower := make(map[NodeID]uint64)
 
-		var totalVotingPower uint64
 		for i := range validators {
-			validators[i] = NodeID(strconv.Itoa(i))
-			votingPower[validators[i]] = stake[i]
-			totalVotingPower += stake[i]
+			nodeId := NodeID(strconv.Itoa(i))
```

Comment:
Let's not only do property-based testing here but also the traditional unit tests.

Reply:
Agreed. Already have one for `MaxFaulty` calculation, I'm going to add the one for `QuorumSize` as well.